### PR TITLE
GitHub comments

### PR DESCRIPTION
**Pull request by silverdrake11**
Feel free to update if you dont like the styling

### CODE REVIEW

**Comment by silverdrake11** [comment]
Conwell was born about 1748 in Moneymore, County Londonderry, Ireland (present-day Northern Ireland), the son of Owen Conwell and his wife, Mary (Keegan) Conwell.[a][1][2] He studied at the Irish College in Paris and was ordained to the priesthood there in November 1776.[2][3] Conwell was well-educated, being fluent in Latin, French, Spanish, and Italian, but had a reputation as a preacher of only modest ability.[4] Around 1785, he contemplated moving to the United States but was concerned about the still uncertain financial state of the new country's parishes, and ultimately decided not to apply for a position there.[3] He was appointed parish priest of Dungannon in 1792 or 1793 and Vicar General of the Archdiocese of Armagh in 1794.[1][5] After the death of Archbishop Richard O'Reilly in 1818, Conwell served as acting Archbishop.[6]

**Comment by silverdrake11** [comment]
Here's the screenshot


Screen Shot 2020-05-06 at 7 02 08 PM

**Comment by silverdrake11** [comment]
@chrisluedtke

File: _layouts/post.html [hunk 1]
```diff
@@ -32,7 +32,8 @@ <h1 class="post-title">{{ page.title | escape }}</h1>
       </div>
 
       {% if page.gh_comments_issue_id %}
-      <h2>Comments</h2>
+      <br></br>
+      <h1>Comments</h1>
```
Comment (chrisluedtke) [inline _layouts/post.html:36]: `h1` feels too large to me

**Pull request merged**